src/main/java/jimmy/mcgymmy/logic/commands/ImportCommand.java

```diff
@@ -39,12 +42,14 @@ public CommandResult execute(Model model) throws CommandException {
             JsonMcGymmyStorage importedMcgymmy = new JsonMcGymmyStorage(filepath);
             Optional<ReadOnlyMcGymmy> readOnlyMcGymmyOptional = importedMcgymmy.readMcGymmy();
             if (readOnlyMcGymmyOptional.isEmpty()) {
-                throw new CommandException(MESSAGE_IMPORT_FOOD_FAILURE);
+                throw new CommandException(String.format(MESSAGE_IMPORT_FOOD_FAILURE, filepath));
```

Comment:
possibly save the formatted string as a variable? since it is reused quite a bit.

Reply:
It changes from each execution of the command so at most it can only be a local variable.
Can make much of a difference imo.